src/features/device-config/download.ts

```diff
@@ -29,7 +29,7 @@ const getApp = async (appId: number, req: Request) => {
 				},
 			},
 		},
-	})) as
+	} as const)) as
```

Comment:
Any reason for keeping this cast?

Reply:
I think the reason for all these casts being kept is because they rely on nested $expand support, which didn't exist at the point I initially wrote the PR